src/main/java/org/broadinstitute/dsm/model/UPSKit.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsm.model;
+
+import lombok.Data;
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.model.ups.UPSPackage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+@Data
+public class UPSKit {
+    // the two shipments differ in tracking id
+    UPSPackage upsPackage; // package with tracking_to_id
+    String kitLabel;
+    Boolean CE_order;
+    String dsmKitRequestId;
+    String externalOrderNumber;
+    String trackingToId;
+    String trackingReturnId;
+    String ddpInstanceId;
+    String hruid;
+
+    private static final Logger logger = LoggerFactory.getLogger(UPSKit.class.getName());
+
```

Comment:
I made a simpler and smaller object than `DdpKit` to use with the new UPS classes